ui/pages/confirm-encryption-public-key/confirm-encryption-public-key.component.js

```diff
@@ -46,21 +46,8 @@ export default class ConfirmEncryptionPublicKey extends Component {
   };
 
   _beforeUnload = async (event) => {
-    const {
-      clearConfirmTransaction,
-      cancelEncryptionPublicKey,
-      txData,
-    } = this.props;
-    const { metricsEvent } = this.context;
+    const { cancelEncryptionPublicKey, txData } = this.props;
     await cancelEncryptionPublicKey(txData, event);
```

Comment:
`cancelEncryptionPublicKey` results in the message being rejected in the background. Since we're doing this automatically on window close now, we don't need to call this anymore.

The same goes for `cancelDecryptMessage` as well, in the `confirm-decrypt-message.component.js` beforeunload handler. Both of these beforeunload handlers are no longer needed.